.travis.yml

```diff
@@ -5,10 +5,7 @@ os:
 language: node_js
 node_js:
   - node
-  - '7'
-  - '6'
-  - '5'
-  - '4'
-  - '0.12'
-  - '0.10'
-secure: "ZPTokHsJs6NtWFhUyG5GH0YnS+eE/6HvfnKZjoynF1uA51+SY2wp4tMoE5y2j2VFDBa0DBeKRChdzDSMhjZQDXqUWNzv+AObdzOlVJqovdQgBcwcENB9bD1MF6ZRSpol7ETi5lDkCH3fpi7RG/gb9HIyb1F63tzo5DXewfXOUlk="
+  - '9'
+  - '8'
+secure: >-
```

Comment:
I wonder if this needs to be recreated to get travis to run properly. Since the error is indicating that it's not finding the username and password, I think the `GITHUB_USERNAME` and `GITHUB_PASSWORD` environment variables aren't getting set correctly.